src/components/RaceCards/RaceCard.tsx

```diff
@@ -0,0 +1,113 @@
+import Link from 'next/link';
+import { motion } from 'framer-motion';
+import styles from './RaceCard.module.scss';
+
+type RaceCardProps = {
+    header: string;
+    url?: string
+    abilityBonuses: Array<any>;
+    traits: Array<any>;
+};
+
+function RaceCard({
+  url = 'races', traits, abilityBonuses, ...props
+}: RaceCardProps) {
+  // Card animations
+  const hoverOn = {
+    rest: {
+      scale: 1,
+      originX: 0,
+      transition: {
+        duration: 0.40,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+    },
+    hover: {
+      scale: 1.1,
+      transition: {
+        delay: 0.40,
+        duration: 0.65,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+      zIndex: 10,
+    },
+  };
+  const headerOn = {
+    rest: {
+      backgroundColor: 'rgba(0, 0, 0, 0.65)',
+      top: 240,
+      transition: {
+        duration: 0.40,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+    },
+    hover: {
+      backgroundColor: 'rgba(0, 0, 0, 0.75)',
+      top: 0,
+      color: '#dec74c',
+      transition: {
+        duration: 0.65,
+        delay: 0.40,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+    },
+  };
+  const contentOn = {
+    rest: {
+      backgroundColor: 'rgba(0, 0, 0, 0.65)',
+      top: 240,
+      transition: {
+        duration: 0.40,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+    },
+    hover: {
+      backgroundColor: 'rgba(0, 0, 0, 0.75)',
+      top: 0,
+      transition: {
+        duration: 0.65,
+        delay: 0.40,
+        ease: 'easeOut',
+        bounce: 0,
+      },
+    },
```

Comment:
I would consider these styles as animations, maybe make an` animations.ts` file and import them that way we arent repeating these in other area's